### ISSUE TITLE
`/eth/v1/validator/duties/attester/{epoch}` takes too long

### ISSUE THREAD

**Issue by mcdee** [Consensys-Incorporated/teku#7562]
### Description

Running the above endpoint with 50,000 validators takes nearly 1 minute to return.  On other beacon nodes it takes ~1 second.

### Steps to Reproduce (Bug)

Run the above endpoint for the current epoch, submitting a body containing 50,000 indices.

**Expected behavior:** return the data in a reasonable timeframe (~1s).
 
**Actual behavior:** takes around 1 minute to respond, which loses the ability to attest for slots early in the epoch.

**Frequency:** Always, with this number of validators.

### Versions (Add all that apply)
* Software version: teku/v23.9.1/linux-x86_64/-privatebuild-openjdk64bitservervm-java-17

**Comment by rolfyone** [comment]
Just trying to replicate this...
I generated a file that was basically 
```
["100", "101"..., "50100"]
```
and sent that with a little test script:
```
#!/usr/bin/bash

if [ -z "$1" ]; then exit 1; fi

curl -X POST \
	-v --trace-time \
	-w "@curl-format.txt" \
	http://localhost:5051/eth/v1/validator/duties/attester/$1 \
 -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d @body.json \
  -o out.json
```
curl-format was just: 
```
     time_namelookup:  %{time_namelookup}s\n
        time_connect:  %{time_connect}s\n
     time_appconnect:  %{time_appconnect}s\n
    time_pretransfer:  %{time_pretransfer}s\n
       time_redirect:  %{time_redirect}s\n
  time_starttransfer:  %{time_starttransfer}s\n
         Speed_upload: %{speed_upload}\n
                     ----------\n
          time_total:  %{time_total}s\n
```
I was mostly concerned where the time was going...

trace-time shows:
```
./test.sh 232986
Note: Unnecessary use of -X or --request, POST is already inferred.
23:48:09.888907 *   Trying 127.0.0.1:5051...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     023:48:09.889409 * Connected to localhost (127.0.0.1) port 5051 (#0)
23:48:09.889475 > POST /eth/v1/validator/duties/attester/232986 HTTP/1.1
23:48:09.889475 > Host: localhost:5051
23:48:09.889475 > User-Agent: curl/7.81.0
23:48:09.889475 > accept: application/json
23:48:09.889475 > Content-Type: application/json
23:48:09.889475 > Content-Length: 439200
23:48:09.889475 >
23:48:09.889735 } [65536 bytes data]
23:48:09.889922 * We are completely uploaded and fine
100  428k    0     0  100  428k      0  28860  0:00:15  0:00:15 --:--:--     023:48:25.130074 * Mark bundle as not supporting multiuse
23:48:25.130116 < HTTP/1.1 200 OK
23:48:25.130137 < Date: Mon, 02 Oct 2023 23:48:09 GMT
23:48:25.130156 < Content-Type: application/json
23:48:25.130175 < Transfer-Encoding: chunked
23:48:25.130192 < Server: Jetty(11.0.15)
23:48:25.130211 <
23:48:25.130229 { [32774 bytes data]
100 8617k    0 8188k  100  428k   534k  28678  0:00:15  0:00:15 --:--:-- 1996k
23:48:25.203686 * Connection #0 to host localhost left intact
     time_namelookup:  0.000017s
        time_connect:  0.000540s
     time_appconnect:  0.000000s
    time_pretransfer:  0.000798s
       time_redirect:  0.000000s
  time_starttransfer:  0.000800s
         Speed_upload: 28678
                     ----------
          time_total:  15.314750s
```

basically most of the 15 seconds were taken receiving data, so that'll be something to do with how we're constructing the response...

15 seconds is still way too long, but just spelling out how i measured.

I'll try to replicate this in a better way so that i can profile and see what's going on.

**Comment by rolfyone** [comment]
fixed in 23.10.0
